### ISSUE TITLE
Corruption Removal

### ISSUE THREAD

**Issue by gjpeebles** [Tirocupidus/TheExiledRPOverhaul#110]
Just an idea right now.

Disable the ability for dancers to remove corruption. In its place, allow priests to use favor in some way to remove it. Crom-followers will just be stuck with it.

Just kidding. Crom-followers might be able to slowly remove it through fighting or something.

**Comment by Shadowcrafting** [comment]
Faith food removes corruption and gives health regency

Dancers grant drama regency... For obvious reasons xD

**Comment by Shadowcrafting** [comment]
Dancers grant a stamina regen buff I ment

**Comment by CraftsmasterKubilai** [comment]
IDEA: Crom worshipers, since they have no priests or "magic food", should "heal" corruption by taking damage. I suggest that since Crom worshipers are inherently at a disadvantage compared to all other religions, the heal effect should be pretty powerful (somewhere in the range of "heal a full bar of corruption for killing 1-2 skeletons"). I also suggest that Crom be a religion players can choose at Character Creation (since it doesn't make any sense for there to be a TERPO feat for Crom given their lack of any other benefits/structures/tools). 

RELATED IDEA: Crom worshipers wielding Hardened Steel weapons (which thematically are obviously supposed to represent "northern steel") should have this effect buffed (1.5X or 2X compared to using any other weapon). 

KEEP DANCER HEAL AT REDUCED/SLOWER RATE: Last, I'd suggest KEEPING the vanilla "dancers remove corruption" mechanic in conjunction with any religion-based change, but nerf the heal effect so that it operates 2X, 3X or even longer - this way, solos or players who for RP reasons oppose the religion can still remove corruption (albeit more slowly.)

**Comment by gjpeebles** [comment]
Possibly add a *very* slow innate corruption reduction so people aren't completely dependent on priests. Or allow them to petition for a DM/admin priest. I just don't want anyone to feel they're stuck with it.

**Comment by CraftsmasterKubilai** [comment]
Oh I forgot: REH was pretty clear that Set and Yog were beings from the Outer Dark, so it doesn't make sense (IMHO) for Yog Meat or Antivenom (their "priest food") to reduce Corruption, only Mitra's Ambrosia. Maybe we could add  a corruption removal effect to Aloe Pots on top of its healing ability, if you're all about making it "food-related"?

**Comment by Shadowcrafting** [comment]
Using prey emote at a shrine allows you to remove corruption at that god's shrine. It also gives you a 24 hour buff where if you attempt to prey at another god's shrine, you gain corruption!

**Comment by CraftsmasterKubilai** [comment]
Re: reducing naturally over time - I'd like it better of the player actually had to DO something to remove it beyond sit and hide, hence the suggestion to keep the dancer effect (just at a reduced speed/rate).

**Comment by CraftsmasterKubilai** [comment]
I like the pray emote thing, but again, if we're being true to the source material then praying at Yog's temple should actually GAIN corruption (Set, too, if we're honest). :P

**Comment by gjpeebles** [comment]
Thematically it can be a different kind of corruption than that which is gained in service to a deity.

**Comment by gjpeebles** [comment]
It's something that physically hinders you, so your god is helping to remove it.

**Comment by Shadowcrafting** [comment]
Yog corruption= hunger debuff but faster healing
Set corruption=  items decay faster, but you are immune to ALL poison

**Comment by CraftsmasterKubilai** [comment]
I suppose... but even the IG lore in the Forbidden City makes it clear that the Alienhead Builders were Set/Yog worshipers, so I always assumed the corruption was still "Outer Dark-y". :P

**Comment by Shadowcrafting** [comment]
I'd say that's more due to what's living in places, undead = not healthy to be around large numbers of them

**Comment by Shadowcrafting** [comment]
Maybe preying to a alter gives you a permanent buff faithful that allows you to gain benefits of worship?

**Comment by CraftsmasterKubilai** [comment]
A permanent buff for just performing an emote in a certain spot feels too powerful (IMHO), but how about a TEMPORARY effect (duration: at least an hour, or possibly "until the sun rises/sets again") instead, like:

Mitra: Buff healing regen rate while the effect is in-place
Set: Add a poison degen to all attacks while effect is in place
Yog: Reduce thirst/hunger rate while the effect is in-place
Ymir: ?? (need to learn more about them, sorry). 

Priests,. however, could have a more powerful, permanent version of the same effect at all times. scaled up based on current priest level in TERPO (since they're more than just a worshipper, their god grants them more powerful versions continuously).

**Comment by gjpeebles** [comment]
These comments are getting more off-topic, but good for my overall religion overhaul I'd like to do at some point.

**Comment by CraftsmasterKubilai** [comment]
Fair enough - is there a Git ticket for that? If not, my last comment may be a good place to start it? Unless you have other ideas that are better, or course... 👍

**Comment by gjpeebles** [comment]
Probably make a new Issue called "Shrine Buffs" or something. I can add it to the [Religion Overhaul](https://github.com/Tirocupidus/TheExiledRPOverhaul/projects/5) project.
